template/web/src/components/SmartWallets/PaymasterClient.ts

```diff
@@ -1,7 +1,17 @@
-import { createPublicClient, http } from 'viem';
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
+import { paymasterActionsEip7677 } from 'permissionless/experimental';
+import { createClient, http } from 'viem';
+import { createPublicClient } from 'viem';
```

Comment:
why two lines to import from `viem`, just do one line.

Reply:
Whoops, fixed!

Reply:
> - [ ] @